components/kyma-environment-broker/internal/migrations/README.md

```diff
@@ -0,0 +1,10 @@
+## Migrations
+
+Whenever we want to execute migration which requires breaking changes, and a proper migration this package can be used to store migration files.
+
+### Best practices
+
+- For each migration which requires breaking changes there should be implemented a rollback which can be used to revert changes in case of any problems
```

Comment:
```suggestion
- For each migration that requires breaking changes, you should implement a rollback that easily reverts changes in case of any problems.
```